vue-frontend/src/components/contact/ContactFormNew.vue

```diff
@@ -0,0 +1,367 @@
+
+export default {
+  data() {
+    return {
+      loaderImage,
+      faCaretDown,
+      faCaretUp,
+      showList: false,
+      name: "",
+      email: "",
+      projectInfo: "",
+      reference: "",
+      invest: "I'll invest",
+      investOptions: [
+        "< USD 50000",
+        "USD 50000 - USD 100000",
+        "USD 100000 - USD 250000",
+        "> USD 250000",
+      ],
+      NDA: false,
+      disableInput: false,
+      showValidationError: false,
+      showSuccessMessagePopup: false,
+      showErrorMessagePopup: false,
+      errorMessage: "Something went wrong on our side",
+      showLoader: false,
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+  },
+  mounted() {
+    document.addEventListener("click", this.closePopUps);
+  },
+  unmounted() {
+    document.removeEventListener("click", this.closePopUps);
+  },
+  methods: {
+    submitForm() {
+      if (
+        this.name === "" ||
+        this.email === "" ||
+        this.projectInfo === "" ||
+        this.reference === "" ||
+        this.invest === "I'll invest"
+      ) {
+        this.showValidationError = true;
+      } else {
+        this.disableInput = true;
+        this.showValidationError = false;
+        this.showLoader = true;
+
+        let formData = {
+          name: this.name,
+          email: this.email,
+          project_info: this.projectInfo ? this.projectInfo : "NA",
+          reference: this.reference,
+          invest: this.invest,
+          nda: this.NDA,
+          contact_type: "",
+        };
+
+        // verify recaptcha
+        grecaptcha.enterprise.ready(() => {
+          grecaptcha.enterprise
+            .execute(import.meta.env.VITE_RECAPTCHA_SITE_KEY, {
+              action: "verify",
+            })
+            .then((token) => {
+              formData.token = token;
+              axios
+                .post(config.API_BASE + "/api/send-contact-mail", formData)
+                .then(() => {
+                  setTimeout(() => {
+                    this.showSuccessMessagePopup = true;
+                    this.showLoader = false;
+                  }, 1000);
+                })
+                .catch((err) => {
+                  if (err.response.status == 401) {
+                    this.errorMessage = "Invalid recaptcha score";
+                  }
+                  this.showErrorMessagePopup = true;
+                });
+            })
+            .catch(() => {
+              this.errorMessage = "Invalid recaptcha score";
+              this.showErrorMessagePopup = true;
+            });
+        });
+      }
+    },
+    closePopUps(e) {
+      const showList = this.$refs["invest-list"];
+      if (showList && !showList.contains(e.target)) {
+        this.showList = false;
+      }
+    },
+    setOption(option) {
+      this.invest = option;
+      this.showList = false;
+    },
+  },
+};
+</script>
+
+<style lang="postcss" scoped>
+.floating-input {
```

Comment:
Can we not apply this css as inline ??

Reply:
I think its used on may places that's why added it here

Reply:
But i think we display repeated css inline only
According to Jimmy say !!